bins/tts/train.py

```diff
@@ -20,6 +23,8 @@ def build_trainer(args, cfg):
         "VITS": VITSTrainer,
         "VALLE": VALLETrainer,
         "NaturalSpeech2": NS2Trainer,
+        "VALLE_V2_AR": VALLE_V2_AR,
```

Comment:
For future development: do not use names like "v2", "v3", etc. Name it based on its features.